### DISCUSSION TITLE
issue with Loading a simulation with ospsuite package

### DISCUSSION THREAD

**Discussion by hainiwen** [Open-Systems-Pharmacology/Forum#1508] [Questions & Problems] [answered]
Hello guys.
I met with this problem when I try to load my Mobi program with ospsuite. 

When I run:
library(ospsuite)
simFilePath <- system.file("extdata", "salbutamol0422.pkml", package = "ospsuite")
sim <- loadSimulation(simFilePath)

And it goes:
Error in rClr::clrCall(simulationPersister, "LoadSimulation", .expandPath(filePath),  : 
  Type:    System.ArgumentException
Message: The string was not recognized as a valid Uri.
Parameter name: inputUri
Method:  System.Xml.XmlReader CreateReader(System.String, System.Xml.XmlParserContext)
Stack trace:
   at System.Xml.XmlReaderSettings.CreateReader(String inputUri, XmlParserContext inputContext)
   at System.Xml.Linq.XElement.Load(String uri, LoadOptions options)
   at OSPSuite.Core.Serialization.Xml.PKMLPersistor.Load[T](String pkmlFileFullPath, IDimensionFactory dimensionFactory, IObjectBaseFactory objectBaseFactory, IWithIdRepository withIdRepository, ICloneManagerForModel cloneManagerForModel)
   at OSPSuite.Core.Serialization.Exchange.SimulationPersistor.Load(String pkmlFileFullPath, IWithIdRepository withIdRepository)
   at OSPSuite.R.Services.SimulationPersister.LoadSimulation(String fileName, Boolean resetIds)

These lines goes well when I run the example codes with Aciclovir.pkml

I wonder where did I miss the point?

Lots of thanks!!!

**Comment by hainiwen** [comment]
I am also confused about the file format to read. So I tried to read both .mbp3 and .pkml by saving the simulation alone as pkml file. Neither one works. Doesn't seem to be problem with the example there though.

**Reply by msevestre** [reply]
You can only reads pkml file. 
It seems to be a problem with your installation. What is the language use on your computer? Where is the file stored?

**Reply by hainiwen** [reply]
I used English as my system language but I use a swedish keyboard (kind of the only thing I can think of as related)....i realize there are issues with path so I put it in C/document, however it still doesn't work?

**Comment by PavelBal** [comment]
system.file("extdata", "salbutamol0422.pkml", package = "ospsuite")

Is used only as an example, it creates a path to a file that is part of the package installation. You have to provide a full path to the pkml file of your simulation.

**Reply by hainiwen** [reply]
yes...but i setwd dictionary as the dictionar that the file is in? I think that's okay?

**Comment by PavelBal** [comment]
No, `system.file("extdata",` always looks in the contents of the package.

**Reply by hainiwen** [reply]
So I should write the path where my pkml file is at?? Could you please be more detail here...I didn't really get it

**Reply by PavelBal** [reply]
Something like "C:/MyDocuments/MyModel.pkml"

**Comment by PavelBal** [comment, marked as answer]
```
simFilePath <- "C:/MyDocuments/MyModel.pkml"
sim <- loadSimulation(simFilePath)
```

**Reply by hainiwen** [reply]
Thanks! It worked!!!!
